default.nix

```diff
@@ -1,6 +1,8 @@
-{ lib ? import <nixpkgs/lib> }:
+{ lib ? import <nixpkgs/lib>
+, rootMountPoint ? "/mnt"
```

Comment:
Should the cli not also expose this?

Reply:
ah yeah, haven't used the cli in a while :D probably should also do some tests with it